lambdatimes/src/components/Content/Tab.js

```diff
@@ -1,22 +1,66 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const ActiveTab = styled.div`
+    display: flex;
+    justify-content: none;
+    align-items: center;
+    flex-direction: row;
+    color: #fff;
+    background-color: #333;
+    margin: 0 5px;
+    padding: 2px 10px;
+    font-size: 12px;
+    letter-spacing: 2px;
+    cursor: pointer;
+    font-weight: bold;
+    background-color: #fff;
+    color: #333;
+    border: 2px solid #333;
+`;
+
+const NotActiveTab = styled.div`
+    display: flex;
+    justify-content: none;
+    align-items: center;
+    flex-direction: row;
+    color: #fff;
+    background-color: #333;
+    margin: 0 5px;
+    padding: 2px 10px;
+    font-size: 12px;
+    letter-spacing: 2px;
+    cursor: pointer;
+    font-weight: bold;
+`;
+
+const Tab = (props) => {
+  /* Using your props, determine if the `tab` prop matches the `selectedTab` prop,
+    if they match, the className should be: 'tab active-tab',
+    if it is not it should just be 'tab'*/
+  if (props.tab === props.selectedTab) {
+    return (
+      <ActiveTab onClick={() => { props.selectTabHandler(props.tab) }}>
+        {props.tab.toUpperCase()}
+      </ActiveTab>
+    )
+  } else {
+    return (
+      <NotActiveTab onClick={() => { props.selectTabHandler(props.tab) }}>
+        {props.tab.toUpperCase()}
+      </NotActiveTab>
```

Comment:
Good approach with the active tab / not active tab solution. Another way to do this is to conditionally add a className to the component for styling based on selection. The approach you took is a lot of repeated code. Let me know if you want some guidance on the simplified version.